tests/configs/k8s_with_node_selectors/kubetools.yml

```diff
@@ -0,0 +1,44 @@
+name: generic-app-with-node-selectors
```

Comment:
Please reconsider the use of `generic-` here... The word does not seem to bring much descriptive value to the test.
I suspect that the word is coming from layers of copy-pasta of tests but originated from a test where it had some form of meaning (i.e. "generic" as opposed to another part of the test that would have been "specific"), and it doesn't seem to me to be the case here.

Reply:
[2885b4c](https://github.com/EDITD/kubetools/pull/161/commits/2885b4c368dd4e58c2b4de1416f8d0dd3166cec2)